plugins/source/aws/resources/services/codepipeline/pipelines_mock_test.go

```diff
@@ -44,6 +44,7 @@ func buildPipelines(t *testing.T, ctrl *gomock.Controller) client.Services {
 	if err := faker.FakeObject(&tags); err != nil {
 		t.Fatal(err)
 	}
+	tags.NextToken = nil
```

Comment:
Why did this line change?

Reply:
Because we weren't handling pagination correctly initially.

Reply:
They shouldn't be. I thought I had a separate (local) branch that I was keeping with these fixes... Looking into it now